src/QsCompiler/CompilationManager/AssemblyLoader.cs

```diff
@@ -27,15 +27,20 @@ namespace Microsoft.Quantum.QsCompiler
     public static class AssemblyLoader
     {
         /// <summary>
-        /// Loads the Q# data structures in a referenced assembly given the Uri to that assembly,
-        /// and returns the loaded content as out parameter.
-        /// Returns false if some of the content could not be loaded successfully,
+        /// Loads the Q# data structures in a referenced assembly given <paramref name="asm" />,
```

Comment:
Nitpicking, I've mostly seen self-closing tags written without a space at the end, like `<paramref name="asm"/>`. (Like shown [here](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/paramref) or [here](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/inheritdoc).)

Reply:
Sounds good! I can easily change all of these automatically with the CSharp doc comment rewriter tool. This is just what C#'s built in XML writer seems to do out of the box for whatever reason.